website/pages/en/index.js

```diff
@@ -58,13 +58,10 @@ const HomeSplash = (props) => {
             <div className="section promoSection">
               <div className="promoRow">
                 <div className="pluginRowBlock">
-                  <Button href="/playground/">Try It Out</Button>
+                  <Button href="/playground/">Try It Out</Button>&nbsp;
                   <Button href={"/docs/" + props.language + "/install.html"}>
                     Get Started
```

Comment:
BTW, to people who see the site for the first time, the difference between "Try It Out" and "Get Started" isn't clear. Both sound like more or less the same. Would be good to think of less confusing captions.

Reply:
What about “Try It Online” and “Installation”?

Reply:
These make more sense.